koku/masu/database/trino_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -285,7 +298,322 @@ GROUP BY coalesce(azure.date, azure.usagedatetime),
+    usage_quantity,
+    currency,
+    pretax_cost,
+    markup_cost,
+    pod_cost,
+    project_markup_cost,
+    pod_usage_cpu_core_hours,
+    pod_request_cpu_core_hours,
+    pod_effective_usage_cpu_core_hours,
+    pod_limit_cpu_core_hours,
+    pod_usage_memory_gigabyte_hours,
+    pod_request_memory_gigabyte_hours,
+    pod_effective_usage_memory_gigabyte_hours,
+    node_capacity_cpu_core_hours,
+    node_capacity_memory_gigabyte_hours,
+    cluster_capacity_cpu_core_hours,
+    cluster_capacity_memory_gigabyte_hours,
+    pod_labels,
+    volume_labels,
+    tags,
+    resource_id_matched,
+    cost_category_id,
+    ocp_source,
+    year,
+    month
+)
+SELECT azure.uuid as azure_uuid,
+    max(ocp.cluster_id) as cluster_id,
+    max(ocp.cluster_alias) as cluster_alias,
+    'Storage' as data_source,
+    CASE
+        WHEN max(persistentvolumeclaim) = ''
+            THEN 'Storage unattributed'
+        ELSE max(namespace)
+    END as namespace,
+    max(ocp.node) as node,
+    max(persistentvolumeclaim) as persistentvolumeclaim,
+    max(persistentvolume) as persistentvolume,
+    max(storageclass) as storageclass,
+    max(azure.resource_id) as resource_id,
+    max(azure.usage_start) as usage_start,
+    max(azure.usage_start) as usage_end,
+    max(nullif(azure.service_name, '')) as service_name,
+    max(azure.instance_type) as instance_type,
+    max(azure.subscription_guid) as subscription_guid,
+    max(azure.subscription_name) as subscription_name,
+    max(nullif(azure.resource_location, '')) as resource_location,
+    'GB-Mo' as unit_of_measure, -- Has to have this unit to show up on ocp on cloud storage endpoint
+    max(cast(azure.usage_quantity as decimal(24,9))) as usage_quantity,
+    max(azure.currency) as currency,
+    max(persistentvolumeclaim_capacity_gigabyte) / max(az_disk.capacity) * max(cast(azure.pretax_cost as decimal(24,9)))  as pretax_cost,
+    (max(persistentvolumeclaim_capacity_gigabyte) / max(az_disk.capacity) * max(cast(azure.pretax_cost as decimal(24,9)))) * cast({{markup}} as decimal(24,9)) as markup_cost, -- pretax_cost x markup = markup_cost
+    cast(NULL as double) as pod_cost,
+    cast(NULL as double) as project_markup_cost,
+    cast(NULL as double) as pod_usage_cpu_core_hours,
+    cast(NULL as double) as pod_request_cpu_core_hours,
+    cast(NULL as double) as pod_effective_usage_cpu_core_hours,
+    cast(NULL as double) as pod_limit_cpu_core_hours,
+    cast(NULL as double) as pod_usage_memory_gigabyte_hours,
+    cast(NULL as double) as pod_request_memory_gigabyte_hours,
+    cast(NULL as double) as pod_effective_usage_memory_gigabyte_hours,
+    cast(NULL as double) as node_capacity_cpu_core_hours,
+    cast(NULL as double) as node_capacity_memory_gigabyte_hours,
+    cast(NULL as double) as cluster_capacity_cpu_core_hours,
+    cast(NULL as double) as cluster_capacity_memory_gigabyte_hours,
+    cast(NULL as varchar) as pod_labels,
+    cast(NULL as varchar) as volume_labels,
```

Comment:
(suggestion) I think we should just remove these NULL selects and remove them from the insert statement. It's very difficult to follow this sql because of how much extraneous stuff is here.

Also, casting a NULL to something just returns a NULL, so I'm not really sure this is necessary anyway.